consul-secure-acl-vault/step5.md

```diff
@@ -0,0 +1,111 @@
+You can verify the Consul server started correctly by checking the logs.
```

Comment:
This looks like a duplicate from step three. I think we should definitely have them check the logs, but probably only need to once before assigning the token.